src/hotspot/share/gc/shared/cardTable.cpp

```diff
@@ -26,13 +26,40 @@
 #include "gc/shared/cardTable.hpp"
 #include "gc/shared/collectedHeap.hpp"
 #include "gc/shared/space.inline.hpp"
+#include "gc/shared/gcLogPrecious.hpp"
+#include "gc/parallel/objectStartArray.hpp"
```

Comment:
This include is in the wrong order and should probably be guarded by `#if INCLUDE_PARALLELGC`.